Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -139,7 +139,7 @@
 
 #endif // !PLATFORM(MAC) && !PLATFORM(MACCATALYST) && !PLATFORM(WATCHOS) && !PLATFORM(APPLETV)
 
-#if !defined(ENABLE_APPLE_PAY_SESSION_V11) && ((PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 110300) || (PLATFORM(IOS) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 140500) || PLATFORM(VISION))
+#if !defined(ENABLE_APPLE_PAY_SESSION_V11) && ((PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 110300) || PLATFORM(IOS) || PLATFORM(VISION))
```

Comment:
I think macOS here is also 2020, and either way, it's best to not touch this line multiple times.